doc/rst/platforms/infiniband.rst

```diff
@@ -170,6 +173,26 @@ configuration output.
 
    export GASNET_IBV_SPAWNER=mpi
 
+As mentioned above, a potential downside of using MPI for launching
+Chapel programs is that the resources that it requires to get the
+program up and running can interfere with those needed by GASNet.  In
+some cases, this can result in negative impacts on performance.  In
+others, it can prevent GASNet from accessing the network resources it
+requires at all.  For example, the following error is an example of
+one in which MPI is preventing GASNet from accessing what it needs
+(albeit using the ``ofi`` conduit rather than the ``ibv`` conduit
+preferred for InfiniBand):
+
+.. code-block:: bash
+
+   *** FATAL ERROR (proc 0): in gasnetc_ofi_init() at /third-party/gasnet/gasnet-src/ofi-conduit/gasnet_ofi.c:1336: fi_endpoint for rdma failed: -22(Invalid argument)
+
+
+For tips and best practices about how to configure/use GASNet to avoid
+such conflicts with MPI, please refer to the section "Build-time
+Configuration" in the [GASNet
+README](https://bitbucket.org/berkeleylab/gasnet/src/master/other/mpi-spawner/README).
```

Comment:
Strictly speaking that link is to "GASNet's README file for mpi-spawner", not "the GASNet README".  
Same in the OmniPath document.

Reply:
Good point.  I've updated this to:

> in the [GASNet README for the MPI spawner]